typing/types.ml

```diff
@@ -454,6 +454,15 @@ let decl_is_abstract decl =
   | Type_abstract _ -> true
   | Type_record _ | Type_variant _ | Type_open -> false
 
+let decl_is_unboxed decl =
```

Comment:
To me `decl_is_unboxed` suggests a bool-returning function; I tend to use `find_*` for option-returning functions. Have you considered calling this function `find_unboxed_type`?

Reply:
Done.